admin/component/admin/users/UserTable.tsx

```
@@ -46,7 +48,7 @@ const UserTable: React.FC<Props> = ({
             {users.map(user => (
               <tr key={user._id} className="hover:bg-gray-50 transition-colors">
                 <td className="whitespace-nowrap py-4 pl-6 pr-3 text-sm text-gray-900">
-                  {user.name}
+                  {user.first_name + ' ' + user.last_name}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)